### PR TITLE
Add comments to model schema (fixes #740)

### LINKED ISSUE #740: Add comments to code
## Feature Request
Some of the features were merged pretty quickly and there was no time to add the comments. (ModelSchema and WebSQL adapter). I need to go back through the code and add adequate comments.

### PR DESCRIPTION
**Pull request by kingsleyzissou**
### Description

I have just gone in and added comments to the ModelSchema for future reference.

### CODE REVIEW

**Review by Eunovo — APPROVED** [review]

File: packages/datastore/datastore/src/ModelSchema.ts [hunk 1]
```diff
@@ -14,28 +14,75 @@ export interface GenetratedModelSchema {
...
+  /** Primary key field flag if the field is
+   * the primary key
+   */
   primary?: boolean;
+  /**
+   * Default value for the field
+   */
   default?: any;
+  /** Flag for if the field should be encrypted
+   * i.e. password field
```
Comment (wtrocki) [inline packages/datastore/datastore/src/ModelSchema.ts:30]:
```suggestion
   * i.e. password field
```

Password field should never be passed here :D

File: packages/datastore/datastore/src/ModelSchema.ts [hunk 2]
```diff
@@ -56,52 +103,100 @@ export class ModelSchema<T = any>{
     this.encrypted = extractEncryptedFields(this.fields, schema.encrypted);
   }
 
+  /**
+   * Method to fill the object with default values
+   *
+   */
   public fill(): void {
```
Comment (wtrocki) [inline packages/datastore/datastore/src/ModelSchema.ts:110]: It is redundant to have this method then? :)

File: packages/datastore/datastore/src/ModelSchema.ts (hunk 1 above)
Comment (kingsleyzissou) [reply packages/datastore/datastore/src/ModelSchema.ts:30]: @wtrocki so the value will never be passed there, but surely at some point the model will have to handle sensitive fields that might need encryption, or not really?

File: packages/datastore/datastore/src/ModelSchema.ts (hunk 2 above)
Comment (kingsleyzissou) [reply packages/datastore/datastore/src/ModelSchema.ts:110]: @wtrocki thinking about it further this method is redundant. We have a default value property though, which is not being used anywhere though.

File: packages/datastore/datastore/src/ModelSchema.ts (hunk 1 above)
Comment (wtrocki) [reply packages/datastore/datastore/src/ModelSchema.ts:30]: Yep. Just remove password as example.

File: packages/datastore/datastore/src/ModelSchema.ts (hunk 2 above)
Comment (wtrocki) [reply packages/datastore/datastore/src/ModelSchema.ts:110]: It is critical for us to implement migrations (this will be required for CLI etc.) Keep it :D

**Pull request merged**